Exiled.CustomItems/API/Features/CustomItem.cs

```diff
@@ -468,9 +468,38 @@ public virtual uint Spawn(IEnumerable<SpawnPoint> spawnPoints, uint limit)
 
                 spawned++;
 
-                Spawn(spawnPoint.Position.ToVector3, out _);
+                if (spawnPoint is DynamicSpawnPoint dynamicSpawnPoint && dynamicSpawnPoint.Location == SpawnLocation.InsideLocker)
+                {
+                    for (int i = 0; i < 50; i++)
```

Comment:
Why looping from `0` to `49`?

Reply:
There are lockers spawned outside of the map, that are given Locker components, but do not have spawnable item chambers. Trying to spawn an item in one of these results in a nullref, so we need to check for that null value, and if it's found, repeat the random selection to find another locker. 

The loop prevents us from looping indefinitely, by limiting the spawning attempts to 50 tries before 'giving up' and not spawning the locker item.

Reply:
So, essentially, spawning the item **up to** 50 items, but the majority of the time not hitting 50 due to funky lockers. I assume it hits enough 'funky lockers' for the 50 attempts to not become an issue of over-spawning?

Reply:
Ok so I'm dumb and forgot a break; at the end. It's supposed to loop until it finds a place to spawn it, looping a max of 50 attempts before giving up.